uwblueprint/jump-math
# backend: Convert Grade Field to Enum

## Notion ticket link

[Convert Grade Field To Enum](https://www.notion.so/uwblueprintexecs/Convert-Grade-Field-To-Enum-631a1b09650d4fc9a4ccff74f488b02e?pvs=4)


## Implementation description
* Refactoring throughout the codebase so Grade Enum is used
* Changed certain options to use Object.keys(Grade) for readability
* Updated the options for AddClassroomModal to use all Grade options
* Updated testCase function to also remove underscores
<img width="1724" alt="Screenshot 2023-03-12 at 1 48 25 AM" src="https://user-images.githubusercontent.com/87676999/224529174-69a95d42-c293-43ed-a08a-65eb7fa0b9e3.png">




Successful Mutation

## Steps to test
1. Ensure all mutations still work
2. Ensure tests are fixed



## What should reviewers focus on?
* It is important that no existing functionality is broken due to this change


## Checklist
